pagopa/io-app
# feat: [IOCOM-882] New DS on a paid payment screen for a cancelled PN message

## Short description
This PR adds the PN paid notice screen with the new DS (for a cancelled PN message).
|old|new|
|-|-|
|![Screenshot 2024-03-22 at 18 27 18](https://github.com/pagopa/io-app/assets/5150343/8b1d8fa9-ff03-4039-bfcf-b286783436a8)|![Simulator Screenshot - iPhone 15 - 2024-03-22 at 18 00 42](https://github.com/pagopa/io-app/assets/5150343/be9b4520-893d-4161-9325-8de925da0f41)|

## List of changes proposed in this pull request
- New `PaidPaymentScreen` screen

## How to test
Using the io-dev-api-server, generate a cancelled PN message with some paid payments. Since the payments section has not been ported to the new DS yet, flip the `isDesignSystemEnabled` value on the PN navigator for the PN message details screen. Select a PN message that is cancelled, select one of its payments and check the new screen.
